examples/binaries/compose/src/lib.rs

```diff
@@ -104,7 +104,7 @@ mod wasm {
         }
 
         async fn call(&self, input: Vec<u8>) -> Result<Vec<u8>, &'static str> {
-            let reply_bytes = msg::send_bytes_and_wait_for_reply(self.handle, &input[..], 0)
+            let reply_bytes = msg::send_bytes_for_reply(self.handle, &input[..], 0)
```

Comment:
This looks like a breaking change. Even if the ABI may not be broken due to the aggressive optimization Rust made the renaming breaks source compatibility for sure: the old programs' source also have to rename calls to successfully compile.

On the other side I don't think we have to care about compatibility at the moment since we haven't stabilized our net yet. However there is an option: mark the old `send_bytes_and_wait_for_reply` as deprecated and implement it with the help of new `send_bytes_for_reply`.

What do you think? @shamilsan @breathx

Reply:
I think there is no need in any compatibility since we have no libs versioning, and especially contracts one.
It's not a big problem atm. But for the first time adding empty fns with `deprecated` status and todo to remove it some time soon seems good. 
On the other hand, we may ask our ecosystem just to prepare PRs with fixes and merge all of this in the same time

Reply:
I vote for a deprecated wrapper as we have many programs with `send_bytes_and_wait_for_reply` in https://github.com/gear-dapps. It is difficult to prepare multiple PRs in several repos simultaneously.

Reply:
or we can just change `#for_reply` https://github.com/gear-tech/gear/blob/3c5fcedae8c04c88f58304f3ea3a41b6c7144b61/gstd/codegen/src/lib.rs#L247 here to `#and_wait_for_reply`

Reply:
The only used function in dApps is `send_and_wait_for_reply` but we can leave both at the moment, it's up to you.

Reply:
ok I'll keep them